src/widget.rs

```diff
@@ -36,6 +37,8 @@ pub trait WidgetExtManual {
 
     fn connect_map_event<F: Fn(&Self, &Event) -> Inhibit + 'static>(&self, f: F) -> u64;
 
+    fn connect_size_allocate<F: Fn(&Self, &RectangleInt) + 'static>(&self, f: F) -> u64;
```

Comment:
It's already possible to refer to the rectangle type as `Allocation` and the generated widget code does.